src/web/mjs/connectors/ScyllaScans.mjs

```diff
@@ -0,0 +1,98 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ScyllaScans extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'scyllascans';
+        super.label = 'Scylla Scans';
+        this.tags = [ 'manga', 'webtoon', 'english', 'scanlation' ];
+        this.url = 'https://scyllascans.org';
+        this.apiUrl = 'https://api.scyllascans.org';
+    }
+
+    async _getMangas() {
+        const operationName = 'works';
+        const query = 'query works($orderBy: String, $sortBy: String, $first: Int, $offset: Int, $languages: [Int], $showHidden: Boolean) {\n  works(orderBy: $orderBy, sortBy: $sortBy, first: $first, offset: $offset, languages: $languages, showHidden: $showHidden) {\n    name\n    language_name\n    stub\n  }\n}';
+        const variables = {
+            "orderBy": "asc",
+            "sortBy": "id",
+            "first": 1000,
+            "offset": 0,
+            "languages": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+            "showHidden": true
+        };
+
+        const data = await this.fetchGraphQL(new URL('/graphql', this.apiUrl), operationName, query, variables);
+        return data.works.map(work => ({
+            id: `/work/${work.language_name}/${work.stub}`,
```

Comment:
Wouldn't `id: work.stub` be enough?

Reply:
Replaced both in _getMangas and _getMangaFromURI